### PR TITLE
New package: poweralertd-0.2.0

### PR DESCRIPTION
**Pull request by maxice8**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/poweralertd/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'poweralertd'
+pkgname=poweralertd
+version=0.2.0
+revision=1
+archs="*"
```
Comment (ericonr) [inline srcpkgs/poweralertd/template:5]: This can simply be omitted.

File: srcpkgs/poweralertd/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=meson
+configure_args="-Dman-pages=enabled"
+hostmakedepends="pkgconf meson scdoc"
```
Comment (ericonr) [inline srcpkgs/poweralertd/template:8]: `pkg-config` is preferred.

File: srcpkgs/poweralertd/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'poweralertd'
+pkgname=poweralertd
+version=0.2.0
+revision=1
+build_style=meson
+configure_args="-Dman-pages=enabled"
+hostmakedepends="pkg-config meson scdoc"
```
Comment (paper42) [inline srcpkgs/poweralertd/template:7]: Explicitly adding meson to hostmakedepeds is unnecessary, the meson build-style already does this.

**Comment by ericonr** [comment]
This is pretty nifty!

**Pull request merged**